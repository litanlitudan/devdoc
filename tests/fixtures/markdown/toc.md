# Table of Contents

[[TOC]]

## Level 2

Purr as loud as possible, be the most annoying cat that you can, and, knock everything off the table prance along on top of the garden fence, annoy the neighbor's dog and make it bark yet love and coo around boyfriend who purrs and makes the perfect moonlight eyes so i can purr and swat the glittery gleaming yarn to him (the yarn is from a $125 sweater). Brown cats with pink ears. Sit and stare under the bed, for kitty power. Jump five feet high and sideways when a shadow moves stick butt in face have a lot of grump in yourself because you can't forget to be grumpy and not be like king grumpy cat. Spread kitty litter all over house kitty poochy proudly present butt to human. Brown cats with pink ears spit up on light gray carpet instead of adjacent linoleum twitch tail in permanent irritation.

### Level 3

Purr as loud as possible, be the most annoying cat that you can, and, knock everything off the table prance along on top of the garden fence, annoy the neighbor's dog and make it bark yet love and coo around boyfriend who purrs and makes the perfect moonlight eyes so i can purr and swat the glittery gleaming yarn to him (the yarn is from a $125 sweater). Brown cats with pink ears. Sit and stare under the bed, for kitty power. Jump five feet high and sideways when a shadow moves stick butt in face have a lot of grump in yourself because you can't forget to be grumpy and not be like king grumpy cat. Spread kitty litter all over house kitty poochy proudly present butt to human. Brown cats with pink ears spit up on light gray carpet instead of adjacent linoleum twitch tail in permanent irritation.

#### Level 4

Purr as loud as possible, be the most annoying cat that you can, and, knock everything off the table prance along on top of the garden fence, annoy the neighbor's dog and make it bark yet love and coo around boyfriend who purrs and makes the perfect moonlight eyes so i can purr and swat the glittery gleaming yarn to him (the yarn is from a $125 sweater). Brown cats with pink ears. Sit and stare under the bed, for kitty power. Jump five feet high and sideways when a shadow moves stick butt in face have a lot of grump in yourself because you can't forget to be grumpy and not be like king grumpy cat. Spread kitty litter all over house kitty poochy proudly present butt to human. Brown cats with pink ears spit up on light gray carpet instead of adjacent linoleum twitch tail in permanent irritation.

##### Level 5

Purr as loud as possible, be the most annoying cat that you can, and, knock everything off the table prance along on top of the garden fence, annoy the neighbor's dog and make it bark yet love and coo around boyfriend who purrs and makes the perfect moonlight eyes so i can purr and swat the glittery gleaming yarn to him (the yarn is from a $125 sweater). Brown cats with pink ears. Sit and stare under the bed, for kitty power. Jump five feet high and sideways when a shadow moves stick butt in face have a lot of grump in yourself because you can't forget to be grumpy and not be like king grumpy cat. Spread kitty litter all over house kitty poochy proudly present butt to human. Brown cats with pink ears spit up on light gray carpet instead of adjacent linoleum twitch tail in permanent irritation.

###### Level 6

Purr as loud as possible, be the most annoying cat that you can, and, knock everything off the table prance along on top of the garden fence, annoy the neighbor's dog and make it bark yet love and coo around boyfriend who purrs and makes the perfect moonlight eyes so i can purr and swat the glittery gleaming yarn to him (the yarn is from a $125 sweater). Brown cats with pink ears. Sit and stare under the bed, for kitty power. Jump five feet high and sideways when a shadow moves stick butt in face have a lot of grump in yourself because you can't forget to be grumpy and not be like king grumpy cat. Spread kitty litter all over house kitty poochy proudly present butt to human. Brown cats with pink ears spit up on light gray carpet instead of adjacent linoleum twitch tail in permanent irritation.



