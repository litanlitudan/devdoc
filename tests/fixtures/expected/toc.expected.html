<h1 id="table-of-contents">Table of Contents</h1>
<p><div class="table-of-contents"><ul><li><a href="#table-of-contents">Table of Contents</a><ul><li><a href="#level-2">Level 2</a><ul><li><a href="#level-3">Level 3</a><ul><li><a href="#level-4">Level 4</a><ul><li><a href="#level-5">Level 5</a><ul><li><a href="#level-6">Level 6</a></li></ul></li></ul></li></ul></li></ul></li></ul></li></ul></div></p>
<h2 id="level-2">Level 2</h2>
<p>Purr as loud as possible, be the most annoying cat that you can, and, knock everything off the table prance along on top of the garden fence, annoy the neighbor's dog and make it bark yet love and coo around boyfriend who purrs and makes the perfect moonlight eyes so i can purr and swat the glittery gleaming yarn to him (the yarn is from a $125 sweater). Brown cats with pink ears. Sit and stare under the bed, for kitty power. Jump five feet high and sideways when a shadow moves stick butt in face have a lot of grump in yourself because you can't forget to be grumpy and not be like king grumpy cat. Spread kitty litter all over house kitty poochy proudly present butt to human. Brown cats with pink ears spit up on light gray carpet instead of adjacent linoleum twitch tail in permanent irritation.</p>
<h3 id="level-3">Level 3</h3>
<p>Purr as loud as possible, be the most annoying cat that you can, and, knock everything off the table prance along on top of the garden fence, annoy the neighbor's dog and make it bark yet love and coo around boyfriend who purrs and makes the perfect moonlight eyes so i can purr and swat the glittery gleaming yarn to him (the yarn is from a $125 sweater). Brown cats with pink ears. Sit and stare under the bed, for kitty power. Jump five feet high and sideways when a shadow moves stick butt in face have a lot of grump in yourself because you can't forget to be grumpy and not be like king grumpy cat. Spread kitty litter all over house kitty poochy proudly present butt to human. Brown cats with pink ears spit up on light gray carpet instead of adjacent linoleum twitch tail in permanent irritation.</p>
<h4 id="level-4">Level 4</h4>
<p>Purr as loud as possible, be the most annoying cat that you can, and, knock everything off the table prance along on top of the garden fence, annoy the neighbor's dog and make it bark yet love and coo around boyfriend who purrs and makes the perfect moonlight eyes so i can purr and swat the glittery gleaming yarn to him (the yarn is from a $125 sweater). Brown cats with pink ears. Sit and stare under the bed, for kitty power. Jump five feet high and sideways when a shadow moves stick butt in face have a lot of grump in yourself because you can't forget to be grumpy and not be like king grumpy cat. Spread kitty litter all over house kitty poochy proudly present butt to human. Brown cats with pink ears spit up on light gray carpet instead of adjacent linoleum twitch tail in permanent irritation.</p>
<h5 id="level-5">Level 5</h5>
<p>Purr as loud as possible, be the most annoying cat that you can, and, knock everything off the table prance along on top of the garden fence, annoy the neighbor's dog and make it bark yet love and coo around boyfriend who purrs and makes the perfect moonlight eyes so i can purr and swat the glittery gleaming yarn to him (the yarn is from a $125 sweater). Brown cats with pink ears. Sit and stare under the bed, for kitty power. Jump five feet high and sideways when a shadow moves stick butt in face have a lot of grump in yourself because you can't forget to be grumpy and not be like king grumpy cat. Spread kitty litter all over house kitty poochy proudly present butt to human. Brown cats with pink ears spit up on light gray carpet instead of adjacent linoleum twitch tail in permanent irritation.</p>
<h6 id="level-6">Level 6</h6>
<p>Purr as loud as possible, be the most annoying cat that you can, and, knock everything off the table prance along on top of the garden fence, annoy the neighbor's dog and make it bark yet love and coo around boyfriend who purrs and makes the perfect moonlight eyes so i can purr and swat the glittery gleaming yarn to him (the yarn is from a $125 sweater). Brown cats with pink ears. Sit and stare under the bed, for kitty power. Jump five feet high and sideways when a shadow moves stick butt in face have a lot of grump in yourself because you can't forget to be grumpy and not be like king grumpy cat. Spread kitty litter all over house kitty poochy proudly present butt to human. Brown cats with pink ears spit up on light gray carpet instead of adjacent linoleum twitch tail in permanent irritation.</p>
